pep-8104.rst

```diff
@@ -0,0 +1,162 @@
+PEP: 8104
+Title: 2023 Term steering council election
+Version: $Revision$
+Last-Modified: $Date$
+Author: Ee Durbin <ee@python.org>
+Sponsor: Brett Cannon <brett@python.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 08-Nov-2022
+
+
+Abstract
+========
+
+This document describes the schedule and other details of the December
+2022 election for the Python steering council, as specified in
+:pep:`13`. This is the steering council election for the 2023 term
+(i.e. Python 3.12).
+
+
+Election Administration
+=======================
+
+The steering council appointed the
+`Python Software Foundation <https://www.python.org/psf-landing/>`__
+Director of Infrastructure, Ee Durbin to administer the election.
+
+
+Schedule
+========
+
+There will be a two-week nomination period, followed by a two-week
+vote.
+
+The nomination period shall be: November 14, 2022 through November 29, 2022 12:00 UTC
+(The end of November 28, 2022 `Anywhere on Earth
+<https://www.ieee802.org/16/aoe.html>`_).
+
+The voting period shall be: December 1, 2022 12:00 UTC through December 16, 2022
+12:00 UTC (The end of December 15, 2022 `Anywhere on Earth
+<https://www.ieee802.org/16/aoe.html>`_).
+
+
+Candidates
+==========
+
+Candidates must be nominated by a core team member. If the candidate
+is a core team member, they may nominate themselves.
+
+Nominees (in alphabetical order):
+
+- TBD
+
+Withdrawn nominations:
+
+- TBD
+
+Voter Roll
+==========
+
+All active Python core team members are eligible to vote. Active status
+is determined as described in :pep:`PEP 13 <13#membership>`
```

Comment:
Since the link is to the _description_ in PEP 13, not PEP 13 itself, I suggest titling it accurately (particularly helpful for a11y), either:

```suggestion
is determined as :pep:`described in PEP 13 <13#membership>`
```

or

```suggestion
is determined as described in :pep:`13#membership`
```

(which will render as "...as described in [PEP 13 § Membership](https://peps.python.org/pep-0013/#membership)."